diagnostics/eulerian_storm_track/eulerian_storm_track.py

```diff
@@ -0,0 +1,213 @@
+else:
+  ##########################################################
+  # Create the necessary directories
+  ##########################################################
+
+  if not os.path.exists(os.environ['WK_DIR']+'/model'):
+    os.makedirs(os.environ['WK_DIR']+'/model')
+  if not os.path.exists(os.environ['WK_DIR']+'/model/netCDF'):
+    os.makedirs(os.environ['WK_DIR']+'/model/netCDF')
+  if not os.path.exists(os.environ['WK_DIR']+'/model/PS'):
+    os.makedirs(os.environ['WK_DIR']+'/model/PS')
+  if not os.path.exists(os.environ['WK_DIR']+'/obs'):
+    os.makedirs(os.environ['WK_DIR']+'/obs')
+  if not os.path.exists(os.environ['WK_DIR']+'/obs/netCDF'):
+    os.makedirs(os.environ['WK_DIR']+'/obs/netCDF')
+  if not os.path.exists(os.environ['WK_DIR']+'/obs/PS'):
+    os.makedirs(os.environ['WK_DIR']+'/obs/PS')
+  
+  ##################################################################
+  # Reading in the necessary data, and computing the daily eddies
+  ##################################################################
+
+  netcdf_filename = os.environ['MODEL_OUTPUT_DIR']+'/'+os.environ['CASENAME']+'.'+os.environ['v850_var']+'.day.nc'
+  if (not os.path.exists(netcdf_filename)):
+    print ('Cannot Find File: ', netcdf_filename)
+
+  # temporarily add the lat_var and lon_var 
+  # since these values seem to be missing 
+  os.environ['lat_var'] = 'lat'
+  os.environ['lon_var'] = 'lon'
+  os.environ['time_var'] = 'time'
+
+  # reading in the model data
+  ncid = Dataset(netcdf_filename, 'r')
+  lat = ncid.variables[os.environ['lat_var']][:]
+  lat.fill_value = np.nan
+  lat = lat.filled()
+  lon = ncid.variables[os.environ['lon_var']][:]
+  lon.fill_value = np.nan
+  lon = lon.filled()
+  time = ncid.variables[os.environ['time_var']][:]
+  time.fill_value = np.nan
+  time = time.filled()
+  v850 = ncid.variables[os.environ['v850_var']][:]
+  v850.fill_value = np.nan
+  v850 = v850.filled()
+  ncid.close()
+
+  # creating the lat and lon in grid format
+  lonGrid, latGrid = np.meshgrid(lon, lat)
+
+  # getting the daily difference X(t+1) - X(t)
+  eddies = est.transient_eddies(v850)
+  
+  ##########################################################
+  # Creating the plot for the different seasons
+  ##########################################################
+
+  # # getting the all year standard deviation average
+  # season = 'all'
+  # time_ind = est.get_time_ind(int(os.environ['FIRSTYR']), time, season=season)
+  # std_dev = est.model_std_dev(diff, time_ind)
+  # out_file = os.environ['variab_dir']+'/eulerian_storm_track/model/PS/%s.%s.ps'%(os.environ['CASENAME'], season)
+  # plotter.plot(lonGrid, latGrid, std_dev, out_file=out_file, levels=np.arange(0,6), extend='max')
+  # out_file = os.environ['variab_dir']+'/eulerian_storm_track/model/%s.%s.png'%(os.environ['CASENAME'], season)
+  # plotter.plot(lonGrid, latGrid, std_dev, out_file=out_file, levels=np.arange(0,6), extend='max')
```

Comment:
Remove unused commented-out code

Reply:
Done, I removed commented-out code from this and some other places. Also fixed the unused local variable and imports.